assets/scss/modules/_datatables.scss

```diff
@@ -12,31 +13,15 @@
   }
 }
 
-#js-select-all {
-  width: 3%;
-}
-
-.controlpanel {
-  background-color: $white;
-  width: 100%;
-  padding: 5px 0;
-
-  &.fixed {
-    position: fixed;
-    top: 0;
-    z-index: 999;
-  }
-}
-
-.controlpanel__items {
-  position: relative;
+.dataTables_filter__block {
```

Comment:
The dataTables_* elements are classes created by the plugin, but the "dataTables_filter__block" is part of the markup data-dom attributes which configures the table.

I'll rename and push the change.